components/layout/LocaleDropdown.js

```diff
@@ -17,7 +17,7 @@ Options.propTypes = {
     languages: PropTypes.arrayOf(
         PropTypes.shape({
             languageCode: PropTypes.string.isRequired,
-            id: PropTypes.string.isRequired,
+            id: PropTypes.number.isRequired,
```

Comment:
any test?

Reply:
@alfredo-dotcms can we do tests on props?

Reply:
@fmontes https://javascriptplayground.com/failing-tests-on-react-proptypes/

Reply:
https://stackoverflow.com/a/31835256

Reply:
https://medium.com/shark-bytes/type-checking-with-prop-types-in-jest-e0cd0dc92d5

I have not tried those solutions